inspirehep/modules/search/__init__.py

```diff
@@ -27,5 +27,8 @@
 from __future__ import absolute_import, print_function
 
 from .ext import INSPIRESearch
+from .query_factory import inspire_query_factory
 
-__all__ = ('INSPIRESearch', )
+IQ = inspire_query_factory()
```

Comment:
I thin there is no need for a proxy because this is a thread-safe object (it's read-only so there shouldn't be any issue of concurrent access).